msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -422,7 +422,24 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
-#### Generate a deep link to a call
+## Generate a deep link to a meeting
+
+You can also create deep link to start or join a meeting when share to meeting button is selected in an app to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage).
+
+When share to meeting button is selected in an app when user is already in a meeting, the app is shared to stage and a permission dialog pops-up where users can give permissions for the participants such as co-editing a document or collaborating with an app.
+
+When user is not in a meeting then user will be redirected to the Teams calendar where user needs to join a meeting or an instant meeting (Meetnow) can be initiated, where user can add participants and they interact with an app.
+
+| Deep link | Format | Example |
+|-----------|--------|---------|
+|To share the app in an ongoing meeting| `msteams:/l/meeting-share?deeplinkId=GUID&fqdn=string&appContext={json}` ||
```

Comment:
We should add details about each query parameter - deepLinkId, fqdn, appContext.